cudax/include/cuda/experimental/__memory_resource/shared_resource.cuh

```diff
@@ -54,42 +51,104 @@ namespace cuda::experimental::mr
 //! Resource wrapper to share ownership of a resource
 //! --------------------------------------------------
 //!
-//! ``shared_resource`` holds a reference counted :ref:`any_resource <cudax-memory-resource-basic-any-resource>`.
-//! This allows the user to pass a resource around with reference semantics while avoiding lifetime issues.
+//! ``shared_resource`` holds a reference counted instance of a memory resource. This allows
+//! the user to pass a resource around with reference semantics while avoiding lifetime issues.
 //!
 //! @endrst
-template <class... _Properties>
```

Comment:
What happened to properties? They enable interfaces to be defined that expect to take shared ownership of an MR with specific properties. I think we need `shared_resource` to work like `resource_ref` with respect to properties.

Reply:
I believe we do not need that anymore, because we are using the explicit `_Resource` type to forward the properties